nixosConfigurations/hetzner-ephemeral-alpha/default.nix

```diff
@@ -275,6 +220,55 @@ in {
     enable = true;
     configDir = {
       "health_alarm_notify.conf" = config.sops.secrets."netdata/health_alarm_notify.conf".path;
+      "go.d/prometheus.conf" = pkgs.writeText "go.d/prometheus.conf" ''
+        jobs:
+        - name: keep-core
+          url: http://127.0.0.1:9601/metrics
+        - name: caddy
+          url: http://127.0.0.1:2019/metrics
+      '';
```

Comment:
Don't know much about netdata, it has prometheus built-in?

Reply:
https://www.netdata.cloud/prometheus-monitoring/

Oh, I reckon these are endpoints **for** sending metrics to prometheus running on.. ?

Reply:
No, these are prometheus collectors. netdata has a plugin to read Prometheus endpoints and provide gauges from them. Nevertheless, this is much easier than setting up Grafana, and netdata also has a JSON endpoint, so it can effectively translate Prometheus to JSON.